be/src/service/internal_service.cpp

```diff
@@ -279,9 +280,26 @@ void PInternalServiceImpl<T>::cancel_plan_fragment(google::protobuf::RpcControll
     LOG(INFO) << "cancel framgent, fragment_instance_id=" << print_id(tid) << ", reason: " << reason_string;
 
     if (request->has_is_pipeline() && request->is_pipeline()) {
-        auto fragment_ctx = starrocks::pipeline::FragmentContextManager::instance()->get(tid);
-        if (fragment_ctx == nullptr) {
-            LOG(INFO) << "Fragment context already destroyed: fragment_instance_id=" << print_id(tid);
+        TUniqueId query_id;
+        if (!request->has_query_id()) {
```

Comment:
When upgrading StarRocKs, we firstly upgrade BE, then upgrade FE. So this check is invalid.